nng.NETCore/Message.cs

```diff
@@ -102,4 +105,57 @@ protected virtual void Dispose(bool disposing)
 
         
     }
+
+    public class Pipe : IPipe
+    {
+        public Pipe(nng_pipe pipe)
+        {
+            NngPipe = pipe;
+        }
+
+        public nng_pipe NngPipe { get; }
+        public int Id => nng_pipe_id(NngPipe);
+
+        public bool GetOptionBool(string option)
```

Comment:
Can you change the "getopt" functions so they're similar to the other classes; return value is `int` and is result of native `nng_*` function.

In `Listener.cs`:
```csharp
public int GetOpt(string name, out bool data)
            => nng_listener_getopt_bool(NngListener, name, out data);
        public int GetOpt(string name, out int data)
            => nng_listener_getopt_int(NngListener, name, out data);
```

Reply:
Should all these methods for types be renamed to the same name `GetOpt`?  In case of `nng_pipe_getopt_int()` and `nng_pipe_getopt_ms()` functions, they share the same signature so that cannot be overloaded under the same method name.

```c
typedef int32_t nng_duration;
int nng_pipe_getopt_int(nng_pipe p, const char *opt, int *ivalp);
int nng_pipe_getopt_ms (nng_pipe p, const char *opt, nng_duration *durp);
```

Reply:
@jake-ruyi Adjusted so that these methods are consistent with other existing classes: f4b656b81f33e16ebf966e4cbb0b9fd2636b8646.